app/main/views/marketplace.py

```diff
@@ -119,6 +124,13 @@ def external_404():
 
 @main.route('/<framework_framework>/opportunities/<brief_id>')
 def get_brief_by_id(framework_framework, brief_id):
+    frameworks = data_api_client.find_frameworks()['frameworks']
+    frameworks = [framework for framework in frameworks if framework['framework'] == framework_framework]
```

Comment:
https://imgflip.com/i/2azn1o

Reply:
So I can insist we have a variable name change here: https://trello.com/c/iqFcVKpd/14-replace-framework-framework-with-framework-family

Reply:
Yeah I did start doing this, then it cascaded a bit with loads of broken views. I think it's easier to just open a separate PR making these changes, so I'll do that now.

Reply:
Have put up PRs to work on that ticket.